app/shared/modals/reservation-rate-report/ReservationsRateReportContainer.js

```diff
@@ -0,0 +1,34 @@
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+
+import uiActions from 'actions/uiActions';
+import { fetchReservationsRateReport } from 'api/actions';
+import ReservationsRateReportModal from './ReservationsRateReport';
+import Selector from './ReservationsRateReportSelector';
+
+UnconnectedReservationsRateReportModalContainer.propTypes = {
+  onHide: PropTypes.func.isRequired,
+  show: PropTypes.bool.isRequired,
+  units: PropTypes.object.isRequired,
+  fetchReservationsRateReport: PropTypes.func.isRequired,
+};
+
+export function UnconnectedReservationsRateReportModalContainer(props) {
```

Comment:
Why is this needed? Can't the component be used directly if this is what is needed? As in just using `ReservationsRateReportModal` ?

Reply:
Good point, this wrapper is not really needed.